types/fhir-kit-client/index.d.ts

```diff
@@ -1,14 +1,19 @@
-// Type definitions for fhir-kit-client 1.1
+// Type definitions for fhir-kit-client 1.5
 // Project: https://github.com/Vermonster/fhir-kit-client
 // Definitions by: Matthew Morrissette <https://github.com/yinzara>
+//                 Vermonster <https://github.com/Vermonster>
```

Comment:
This looks like an org -- things generally work better with users rather than an org.